config/locales/to_change_everything/pt.yml

```diff
@@ -0,0 +1,305 @@
+
+                    <p>Nada dura para sempre; e isso serve para impérios e civilizações também. Mas o que poderia tomar o lugar dessa ordem? Podemos imaginar uma ordem sem a premissa de uma vida dividida entre o que é legítimo e o que é ilegítimo, legal ou criminoso, comandantes e comandados? O que poderia ser o nosso último crime?</p>
+
+                    <p>E já podemos sentir a lei ruindo ao nosso redor.</p>
+        image_name: lastcrime
+        image_alt: a protestor with his face concealed in the act of throwing a lit
+          molotov cocktail during a protest, in front of a huge concrete wall
+      anarchy:
+        first_html: |-
+          <p><span class="green black">Anarquia</span> é o que acontece quando a ordem não é imposta pela força. É a liberdade: o processo de reinventar a nós e nossas relações continuamente.</p>
+                  <p>Qualquer processo que funciona organicamente – uma floresta tropical, um círculo de amizades, seu próprio corpo – é uma harmonia anárquica que persiste por meio de constante mudança. O controle de cima para baixo, por outro lado, só pode ser mantido pela contenção e pela coerção: a disciplina precária das salas de aula, as fazendas industriais onde pesticidas e herbicidas protegem fileiras estéreis de milho geneticamente modificados, a hegemonia frágil de uma superpotência.</p>
+        second_html: |-
+          <p><span class="pink black">Anarquismo</span> é a ideia de que todas as pessoas têm o direito à completa autodeterminação. Nenhuma lei, nenhum governo ou processo de decisão é mais importante que as necessidades e os desejos de seres humanos reais. As pessoas devem ser livres para moldarem suas relações de acordo com suas satisfações mútuas e defenderem a si mesmas quando se sentirem aptas a isso.</p>
+                    <p>Anarquismo não é um dogma ou um mapa. Não é um sistema que supostamente funcionaria bastando que fosse aplicado corretamente, como a democracia, nem é um objetivo para ser realizado em um futuro muito distante, como o comunismo. É uma forma de agir e nos relacionar que podemos pôr em prática imediatamente. Ele oferece questionamentos que podemos colocar em qualquer situação. Ao invés de questionar se a forma como se desenrola uma ação é legal, ou violenta, ou bondosa, podemos perguntar simplesmente: como ela distribui o poder?</p>
+        third_html: |-
+          <p><span class="blue black">Anarquistas</span> se opõem a toda forma de hierarquia – toda prática que concentra poder na mão de poucas pessoas, todo mecanismo que nos põe distantes de nosso potencial. Contra os sistemas fechados, apreciamos o desconhecido frente a nós, o caos interior através do qual somos capazes de ser livres.</p>
+                    <img src="https://cloudfront.crimethinc.com/tce/images/circle-a-logo-hands-white-large@2x.png">
```

Comment:
remove the `<img>` from `third_html` (it is already in `_show.html.erb`

<img alt='screen shot 2018-12-01 at 3 51 12 pm' src='https://user-images.githubusercontent.com/13190980/49334105-e072b400-f581-11e8-9105-2eb3fda0a007.png' style="max-width:50%;">